### ISSUE TITLE
Transfer learning and fine-tuning tts

### ISSUE THREAD

**Issue by ToiYeuTien** [jaywalnut310/vits#191]
Hi everybody !
I have trained on my computer a Vietnamese female voice model in 500k steps. and I found the voice quite clear. I want to train another male Vietnamese voice.
I learned there is a training method based on a previously trained model, which will shorten the training time.
Can someone help me with that method.
Thank you !

**Comment by CavidanZ** [comment]
Hello. Not sure if a reply this late will help, but that is simply known as transfer learning. You take your first model's checkpoint, pass it as the pre_trained model, and warm start from that point. This will ensure that your model now has the new speaker's voice as well as benefitting from the previous training.

**Comment by ToiYeuTien** [comment]
> Hello. Not sure if a reply this late will help, but that is simply known as transfer learning. You take your first model's checkpoint, pass it as the pre_trained model, and warm start from that point. This will ensure that your model now has the new speaker's voice as well as benefitting from the previous training.

Hello, thank you for your response. 
I understand that to fine-tune the model in such a way, I just need to replace the audio files and metadata of the new model in the location of the old model, and continue training, right? I would appreciate your feedback!

**Comment by CavidanZ** [comment]
Yes. I have done it with tacotron 2 model, and it for sure works. Basically you would do the training just as in the first time: you get your audio dataset ready, and give the model your new audios and the audios' transcriptions. 
1) Just the only difference is you choose the pre trained model to be your previously trained model's checkpoint.
2) You make use of warm starting. It should be like a parameter in hparams that you set to TRUE. 
3) One more thing: do not change the original batch size. At least in tacotron 2 whenever I changed batch size it printed out errors.
